pkg/api/v1/conversion_test.go

```diff
@@ -68,3 +71,33 @@ func TestPodSpecConversion(t *testing.T) {
 		}
 	}
 }
+
+func TestListOptionsConversion(t *testing.T) {
+	testCases := []versioned.ListOptions{
+		{},
+		{ResourceVersion: "1"},
+		{LabelSelector: "a=b,c=d", FieldSelector: "a=b,c!=d", ResourceVersion: "5"},
+	}
+
+	for _, test := range testCases {
```

Comment:
Yes - I verified it. I think the best verification is to use the old format in client and a new format in the server - and I'm doing exactly this in #17369